hudi-common/src/main/java/org/apache/hudi/common/fs/HoodieWrapperFileSystem.java

```diff
@@ -79,8 +81,10 @@ public static void setMetricsRegistry(Registry registry, Registry registryMeta)
     METRICS_REGISTRY_META = registryMeta;
   }
 
+  public static final int MIN_IO_BUFFER_SIZE = 1 * 1024 * 1024; // 1 MB
```

Comment:
Can you add javadocs on why we need 2 different buffer sizes here ?

Reply:
Added